jms/serializer-bundle/2.0/config/packages/jms_serializer.yaml

```diff
@@ -2,12 +2,9 @@ jms_serializer:
     visitors:
         xml:
             format_output: '%kernel.debug%'
-#    metadata:
-#        auto_detection: false
-#        directories:
-#            any-name:
-#                namespace_prefix: "My\\FooBundle"
-#                path: "@MyFooBundle/Resources/config/serializer"
-#            another-name:
-#                namespace_prefix: "My\\BarBundle"
-#                path: "@MyBarBundle/Resources/config/serializer"
+    metadata:
+        auto_detection: true
```

Comment:
do not remember if `auto_detection: true` and `directories` work well together

Reply:
they work together, i checked
see https://github.com/schmittjoh/JMSSerializerBundle/blob/master/DependencyInjection/JMSSerializerExtension.php#L130

Reply:
great! I forgot about it! :)